<launch>
  <node pkg="balance_bot" name="motorA" type="dcmotor_node.py">
  <param name="I1" value="14" />
  <param name="I2" value="15" />
  <param name="pwm" value="18" />
  <param name="encA" value="7" />
  <param name="encB" value="8" />
  <remap from="current_position" to="phiA"/>
  </node>
  <node pkg="balance_bot" name="motorB" type="dcmotor_node.py">
  <param name="I1" value="23" />
  <param name="I2" value="24" />
  <param name="pwm" value="19" />
  <param name="encA" value="11" />
  <param name="encB" value="9" />
  <remap from="current_position" to="phiB"/>
  </node>
</launch>
